extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibConfig.java

```diff
@@ -98,4 +98,10 @@
      */
     @ConfigItem
     public Optional<String> baseRegistryPassword;
+
+    /**
+     * The ports to expose
+     */
+    @ConfigItem(defaultValue = "8080")
+    public List<Integer> ports;
```

Comment:
How about we make it `Optional<List<Integer>> ports` in order to keep the same default behavior as before?

Reply:
What is the current behaviour? The dockerfiles by default expose 8080 but it does not look like jib exposes anything?

Reply:
Yes exactly, currently jib doesn't export anything.

Reply:
So can you actually use it for a HTTP based app?

Reply:
Yeah, because at runtime you can do `-p 8080:8080` (personally I always do this anyway)

Reply:
Ah, I also always do this as well. I assumed you needed both. What about kube? The vague complaint I got was that the lack of a port was causing problems.

Either way even though it is not 100% backwards compatible I think that making the docker and jib builds equivalent is probably worth the minor compat concern.

Reply:
It should be perfectly fine in Kube (I've run it multiple times). 

But I guess it's better to have docker and jib be equivalent than keep backward compatibility.

Reply:
We can probably expose this port automatically (without requiring a user to specify it), but let's leave that for later if anyone ask for it